queries.sql

```diff
@@ -49,3 +49,51 @@ SELECT O.full_name, COUNT(A.name) FROM animals A
 JOIN owners O  ON A.owner_id  = O.id
 GROUP BY O.full_name ORDER BY COUNT(A.name)
 DESC LIMIT 1;
+
+SELECT A.name  FROM animals A
+JOIN visits V ON A.id = V.animals_id 
+JOIN vets T ON V.vets_id = T.id
+WHERE T.name LIKE 'William Tatcher'
+ORDER BY V.date_of_visit ASC LIMIT 1;
```

Comment:
- [ ] Great job working on the quires so far but I think here the result should be ` blossom` since the last animal seen by William Tatcher is `Blossom'` in date `2021-01-11` so kindly fix the query to have the right result 🤩✨🙏🏻

## I see you could just add desc instead of asc since we need the last animal 💪🏻✨